packages/tupaia-web/src/components/CheckboxField.tsx

```diff
@@ -31,7 +42,7 @@ export const CheckboxField = ({ name, label, required, options = {}, ...props }:
       required={required}
       color="primary"
       error={!!errors[name]}
-      helperText={errors[name]?.message}
+      helperText={isArray ? '' : errors[name]?.message}
```

Comment:
This is an ok solution but I wonder if we would do it without needing to make the CheckboxField aware of the other CheckboxList field. Perhaps we can just set helperText={null} in the CheckboxList field instead?